Library/Homebrew/bintray.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
+
+require "utils/curl"
+require "json"
+
+class Bintray
+  API_URL = "https://api.bintray.com"
+
+  def inspect
+    "#<Bintray: user=#{@bintray_user} org=#{@bintray_org} key=***>"
+  end
+
+  def initialize(user: nil, key: nil, org: nil, clear: true)
```

Comment:
Sorry to contradict myself but what about:
```suggestion
  def initialize(user: ENV["HOMEBREW_BINTRAY_USER"], key: ENV["HOMEBREW_BINTRAY_KEY"], org: "homebrew", clear: true)
```